spec/factories/service_order.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
-  factory(:service_order) do
+  factory(:service_order, :class => :ServiceOrderCart) do
     name { "service order" }
     state { "ordered" }
```

Comment:
Is the default for these really "ordered" ? the api says the default state for a service order is 'cart'